specification/quota/resource-manager/Microsoft.Quota/preview/2021-03-15-preview/quota.json

```diff
@@ -342,12 +456,12 @@
         }
       }
     },
-    "/providers/Microsoft.Quota/quotaLimitProviders": {
+    "/providers/Microsoft.Quota/quotaProviders": {
```

Comment:
This API is not behind a feature flag in your published RP manifest. A name change here is a breaking change as the API is technically public. Please discuss this with the breaking change review folks

Reply:
This APIs scope is not limited to user, As it lists what RPs are supported and what dimensions are required for the RP.
I can restore the API name.

Reply:
Restored the name to quotaLimitProviders